### PR TITLE
Updates to datainfo.properties file

### PR DESCRIPTION
**Pull request by jmcinerney**
Cleaning up properties and setting default dev fields

### CODE REVIEW

File: web/src/main/resources/datainfo.properties [hunk 1]
```diff
@@ -14,25 +14,13 @@
 dbType=postgres
 dbUser=clinica
 dbPass=clinica
-db=oc_test_auth0
-#db=oc_test_auth0
-#db=qe-multischema
-#db=qe-ocqe-ocbridge
+db=runtimeDB
```
Comment (svadla-oc) [inline web/src/main/resources/datainfo.properties]: maybe call the database "openclinica"?

Comment (jmcinerney) [reply web/src/main/resources/datainfo.properties]: Updated with this change

**Comment by svadla-oc** [comment]
jenkins retest this please

**Comment by svadla-oc** [comment]
👍

**Pull request merged**